WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -1215,15 +1215,28 @@ - (void)handleFollowConversationButtonTapped
     self.postHeaderView.isSubscribedToPost = newIsSubscribed;
 
     // Define success block
-    void (^successBlock)(void) = ^void() {
-        NSString *title = newIsSubscribed
-            ? NSLocalizedString(@"Successfully subscribed to the comments", @"The app successfully subscribed to the comments for the post")
-            : NSLocalizedString(@"Successfully unsubscribed from the comments", @"The app successfully unsubscribed from the comments for the post");
-
-        dispatch_async(dispatch_get_main_queue(), ^{
-            [generator notificationOccurred:UINotificationFeedbackTypeSuccess];
-            [weakSelf displayNoticeWithTitle:title message:nil];
-        });
+    void (^successBlock)(BOOL taskSucceeded) = ^void(BOOL taskSucceeded) {
+        if (taskSucceeded == NO) {
+            NSString *title = newIsSubscribed
+                ? NSLocalizedString(@"Unable to follow conversation", @"The app failed to subscribe to the comments for the post")
+                : NSLocalizedString(@"Failed to unfollow conversation", @"The app failed to unsubscribe from the comments for the post");
```

Comment:
@ScoutHarris this is not the final form of the error handling or the error messaging. I will be adding a second PR for that.

Reply:
Ah gotcha!